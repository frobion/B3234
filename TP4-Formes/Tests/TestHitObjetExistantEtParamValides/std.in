R Rectangle1 1 3 6 0
HIT Rectangle1 3 2
LIST
EXIT

