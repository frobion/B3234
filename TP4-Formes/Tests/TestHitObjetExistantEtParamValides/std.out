OK
YES
R Rectangle1 1 3 6 0


